### ISSUE TITLE
Growing Mushrooms

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-permaculture#10]
5 Things I Wish I Knew Before I Started Growing Mushrooms For A Living: https://youtu.be/Faq0w-ArCnU
[![image](https://user-images.githubusercontent.com/194400/196350100-4fa96661-bb2d-450e-a8a7-a479ff90031f.png)](https://youtu.be/Faq0w-ArCnU)

**Comment by nelsonic** [comment]
A Guide To Mushroom Substrates: https://youtu.be/gmyMyfMPoTI
[![image](https://user-images.githubusercontent.com/194400/196721418-49ed41a4-5a45-4827-b750-0f45ce99c5b5.png)](https://youtu.be/gmyMyfMPoTI)

**Comment by nelsonic** [comment]
Top 20 Types Of Mushrooms (& Their Uses): https://youtu.be/aGHIz2s0MEY
[![image](https://user-images.githubusercontent.com/194400/196731845-f8514963-bc07-4560-9fbf-75c088e5190b.png)](https://youtu.be/aGHIz2s0MEY)

**Comment by nelsonic** [comment]
Starter Guide To Mushroom Farming: https://youtu.be/icKeO-kyiGk

**Comment by nelsonic** [comment]
I've found myself _watching_ other people go foraging for mushrooms: https://youtu.be/EcvE9Lbj19c`
![image](https://user-images.githubusercontent.com/194400/212469879-41ec58e4-e212-43b3-a5eb-d97c6b607fb2.png)

But the most I've ever foraged was a few [**`Macrolepiota procera`**](https://en.wikipedia.org/wiki/Macrolepiota_procera) in Gerês that we made into soup. But never been with an expert. Need `new` friends. 💭

**Comment by carolrmc** [comment]
So would need to learn this! Only one I’m sure of after watching several
foraging  videos!

**Comment by nelsonic** [comment]
Not at all relevant to growing _edible_ mushrooms ... 🍄 
But absolutely fascinating: `Cordyceps`: https://en.wikipedia.org/wiki/Cordyceps
[![cordyceps](https://user-images.githubusercontent.com/194400/215251983-7144de59-f099-4505-b4d8-b3685476b8ca.png)](https://youtu.be/XuKjBIBBAL8)

The [genus](https://en.wikipedia.org/wiki/Genus) has a worldwide distribution and most of the approximately 600 species[[3]](https://en.wikipedia.org/wiki/Cordyceps#cite_note-3) that have been described are from Asia (notably [Nepal](https://en.wikipedia.org/wiki/Nepal), [China](https://en.wikipedia.org/wiki/China), [Japan](https://en.wikipedia.org/wiki/Japan), [Bhutan](https://en.wikipedia.org/wiki/Bhutan), [Korea](https://en.wikipedia.org/wiki/Korea), [Vietnam](https://en.wikipedia.org/wiki/Vietnam), and [Thailand](https://en.wikipedia.org/wiki/Thailand)). Cordyceps species are particularly abundant and diverse in humid temperate and tropical jungles. 

So we're unlikely to encounter them in `PT` ... 💭 

Reminded of this via: https://www.bbc.com/news/health-64402102 
Fungi are _awesome_! 🤯 

But what the heck is this? https://www.prozis.com/pt/pt/prozis/cordyceps-sinensis-60-caps

<img width="1333" alt="image" src="https://user-images.githubusercontent.com/194400/215252185-68b8f4da-439c-4b9d-8458-740ef5634727.png">

The picture on the bottle is 
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/194400/215252230-384a9956-cb3b-48c0-8e13-e6766159e7ee.png">

Used in "Traditional Chinese Medicine" indeed ... 🙄 
Is there nothing these people wont sell for a quick buck? 🤷‍♂️ 🤦‍♂️ 

https://www.healthline.com/nutrition/cordyceps-benefits ... keeping an open mind.
But Scientists in China developing synthetic versions of known pathogens ... 💭 
Isn't that how we all got **`Covid`** - allegedly ... - https://www.theguardian.com/commentisfree/2022/nov/03/lab-leak-theory-report-propublica-vanity-fair-is-it-reliable

is it _reasonable_ to think that another pandemic is _possible_ in our life times? 🤷‍♂️ 
and if so is it possible that it could be **_considerably_ worse** than **`Covid-19`** ... 😷 
With the acceleration of BioTech and advancements in AI-assisted research 🥼 🔬 🤖 
is it only a matter of _time_ before someone discovers or _creates_ a synthetic `Cordyceps` that can infect humans? ⏳  
And if that happens how long before some mad person _weaponises_ it? 💣 
https://en.wikipedia.org/wiki/List_of_totalitarian_regimes

Should we be prepping? https://github.com/dwyl/phase-three/issues/118 or is it paranoia? 💭

**Comment by nelsonic** [comment]
How to Grow Mushrooms in your Annual or Permaculture Garden: https://youtu.be/QJuh7-05ilo

**Comment by nelsonic** [comment]
Today I bought a Mushroom growing kit at the garden centre section of Leroy Merlin:
![C65FE036-262E-4B66-8E09-87F2A1DD7F31](https://user-images.githubusercontent.com/194400/215886499-1709958e-ff7c-4598-a82e-13e60a62bfa9.jpeg)
![26780646-F1E6-4223-9D0A-18DF0CFB7822](https://user-images.githubusercontent.com/194400/215886507-9a98a61c-ae2e-4379-a811-8c57d0851630.jpeg)
![AAE00233-3BC1-44A8-9BA9-73C195923E06](https://user-images.githubusercontent.com/194400/215886509-c3b792a2-f963-46f4-8ff0-d610699cef50.jpeg)

one of the kits was already fruiting out the side. 🍄

**Comment by nelsonic** [comment]
Going to follow the instructions and see I can get it to work in cellar. It’s on of the rooms in the house where we have a dehumidifier running 24/7 and take out 5L of water each day. It’s mega humid. Which I read is a *good* thing for mushrooms. 🤷‍♂️

**Comment by nelsonic** [comment]
This is such a fascinating topic of study/research:

“ Mushrooms as Medicine with Paul Stamets”: https://youtu.be/7agK0nkiZpA

**Comment by carolrmc** [comment]
They like humidity and warmth, like a compost heap warmth. We had a
mushroom farm nearby as a teen and it was warm and humid in the barn where
they grew. They grew them on horse manure🤪

**Comment by nelsonic** [comment]
Yeah, humid and warm for sure. Might put our kit in kitchen under the sink where there is a bit more heat but still quite dark and plenty humid. 💭

**Comment by nelsonic** [comment]
![93E46251-C487-4BD7-B6F0-56CC07EC78E0](https://user-images.githubusercontent.com/194400/216763226-8c3d32d8-a0a2-40b0-9431-975f905ab0cf.jpeg)

https://youtu.be/5cATnedwHP4
this is very interesting 💭🍄

**Comment by nelsonic** [comment]
This is one of the most fascinating mini-documentaries I've watched in a while: https://youtu.be/jM5jf-8ekVA

**Comment by carolrmc** [comment]
Fascinating!

**Comment by nelsonic** [comment]
@carolrmc thought you might find it interesting. Fungi are so awesome, right?! 🤯 
I feel like I learned exactly _zero_ about them in "school";
wish I still had my text GCSE text books to flick through and confirm. 

But knowing what I know now, 
I wish I could do a biology degree so that I could study them for a PhD!! 

If only there was a way children could learn awesome things in school. 
Instead of being forced to stick to a curriculum designed by bureaucrats ... 

I feel like the _only_ way of ensuring that children are _inspired_ and never lose their curiosity
is to start my own school that _focuses_ on it. 💭

**Comment by carolrmc** [comment]
You’d better get started - it’ll be on top of you before you know it!

**Comment by nelsonic** [comment]
How Japanese Farming Millions of Shiitake Mushroom in Forest : https://youtu.be/DdXJLahuv-g
